graphs/bfs.py

```diff
@@ -26,13 +28,12 @@
 }
 
 
-def bfs(graph, start):
+def bfs(graph: Dict, start: str) -> Set[int]:
     """
     >>> ''.join(sorted(bfs(G, 'A')))
     'ABCDEF'
     """
-    explored, queue = set(), [start]  # collections.deque([start])
-    explored.add(start)
+    explored, queue = set(start), [start]  # collections.deque([start])
```

Comment:
`explored` would be different if `start` was more than a single character.

Reply:
So... how to fix it?